src/inject/GPMInject/interface/toast.js

```diff
@@ -0,0 +1,26 @@
+import { remote } from 'electron';
+
+Emitter.on('toast:pauseAfter', () => {
+  const toast = document.createElement('paper-toast');
+  toast.setAttribute('text', `Pausing after this song.`);
+  toast.duration = 0;
+  toast.noCancelOnOutsideClick = true;
+  const disableButton = document.createElement('a');
+  disableButton.innerHTML = 'Don\'t pause after this song';
+  disableButton.style.color = '#E53935';
+  disableButton.style.margin = '0';
+  disableButton.style.padding = '0';
+  disableButton.style.marginLeft = '10px';
+  disableButton.style.cursor = 'pointer';
+  disableButton.addEventListener('click', (e) => {
+    remote.getGlobal('PlaybackAPI')._setPauseAfter(false);
+    e.preventDefault();
+    return false;
+  });
+  toast.appendChild(disableButton);
+  document.body.appendChild(toast);
+  toast.show();
+  Emitter.on('toastHide:pauseAfter', () => {
+    toast.hide()
```

Comment:
We're hiding the toast here, but the toast is being created via new DOM element, so the old one is never destroyed. Currently I have 3 paper-toasts. Can we optimize this to reuse the same element?